src/aws-cpp-sdk-core/include/smithy/client/features/RecursionDetection.h

```diff
@@ -0,0 +1,64 @@
+/**
+* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#pragma once
+
+#include <iomanip>
+#include "aws/core/platform/Environment.h"
```

Comment:
last nit ill leave on it, give a pass for `""` includes, we likely should just add a clang tidy task to CI to check for this.